File: regx/find_and_return_boolean.cpp
#include <iostream>
#include <regex>
#include <string>

using namespace std;

int main() {
    string pattern_string = "Lee";
    string string_be_search = "An Lee is a famous director";

    regex re(pattern_string);
    
    if (regex_match(string_be_search, regex("Lee"))) {
        cout << "String patter is found!";
    } else {
        cout << "String patter is not found!";
    }

}

File: regx/Find_and_return_boolean.java
import java.util.regex.*;

public class Find_and_return_boolean {
    public static void main(String[] args) {
        String pattern_string = "\\w \\w";
        String string_be_search = "An Lee is a famous director";

        Pattern re = Pattern.compile(pattern_string);
        Matcher mt = re.matcher(string_be_search);

        if(mt.find()) {
            System.out.println("String patter is found!");
        } else {
            System.out.println("String patter is not found!");
        }
    }

}